roles/pulp-webserver/templates/nginx.conf.j2

```diff
@@ -85,7 +85,19 @@ http {
 
         location / {
             # checks for static file, if not found proxy to app
-            try_files $uri =404;
+            try_files $uri @proxy_to_api;
+        }
+
+        location @proxy_to_api {
```

Comment:
Does not having a slash here match against all urls?

I had imagined this would replace the `location / {}` block above. What do you think?

Reply:
Oh I see, yes I think this block should be merged with the local / block above

Reply:
I was unsure if I had to do it on `location / {}` block or if I had to follow how it was before:
https://github.com/pulp/pulp_installer/blob/42e08bb4fd41a1402ad8a912a7af1920b83f3cdc/roles/pulp-webserver/templates/nginx.conf.j2#L65-L83